PocketKit/Sources/PocketKit/Collections/CollectionViewModel.swift

```diff
@@ -88,24 +90,8 @@ class CollectionViewModel: NSObject {
         trackScreenView()
     }
 
-    var title: String {
-        collection.title ?? ""
-    }
-
-    var authors: [String] {
-        ((collection.authors?.compactMap { $0 as? CollectionAuthor }) ?? [CollectionAuthor]()).map { $0.name }
-    }
-
-    var storiesCount: Int? {
-        collectionController.fetchedObjects?.count
-    }
-
-    var intro: Markdown? {
-        collection.intro
-    }
-
     var item: Item? {
-        return collection.item
+        return collection?.item
```

Comment:
we discussed on slack that due to this being `nil` actions are no longer working, and ensuring this is not nil fixes the overflow issue we saw

Reply:
we discussed this in a call so no longer a valid comment. thanks for the fix!